src/test/kotlin/com/exactpro/th2/check1/rule/RuleFactoryTest.kt

```diff
@@ -34,8 +39,11 @@ import com.nhaarman.mockitokotlin2.verify
 import io.reactivex.Observable
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertAll
+import org.junit.jupiter.api.assertDoesNotThrow
 import org.junit.jupiter.api.assertThrows
+import java.time.Instant
 import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
 
 class RuleFactoryTest {
```

Comment:
Is there a test that checks the exception is thrown when the checkpoint does not contain the requested session alias?

Reply:
Yup. The first test with the name `failed rule creation because one of required fields is empty` throws in this case

Reply:
You should verify the exception message then. Could you please add it?

Reply:
Sure. Also covered all cases where a rule cannot be created